scripts/post.py

```diff
@@ -29,8 +29,12 @@
     clean_data = {k: [cleaner.simplify_separators(cleaner.cleanup(x)) for x in v] for k,v in raw_data.items()}
 
     request = pr.PhonetoqueRequest(config)
-    request.pronunciations = clean_data
-    request.prepare_data()
+    # request.pronunciations = clean_data
+    # request.prepare_data()
     #print(request.pronunciations)
 
-    request.post_all()
\ No newline at end of file
+    # request.post_all_words()
+
+    request.get_all_syllables()
+    request.get_max_syllables()
+    request.post_all_syllables()
```

Comment:
I would separate this into separate scripts, one for words and one for syllables, even if it means copying most of this code